recipe/meta.yaml

```
@@ -27,6 +27,9 @@ build:
     - snowflake-dump-ocsp-response-cache = snowflake.connector.tool.dump_ocsp_response_cache:main
     - snowflake-dump-certs = snowflake.connector.tool.dump_certs:main
     - snowflake-export-certs = snowflake.connector.tool.export_certs:main
+  rpaths:
+    - lib/
+    - $SP_DIR/pyarrow
```

I'm sorry I gave imprecise instructions initially. Please use:

```suggestion
    - {{ SP_DIR }}/pyarrow
```

---

Ah! Ok trying this.

---

There are still runtime segfaults on Linux despite this 🤔